src/main/java/health/ere/ps/service/muster16/parser/rgxer/delegate/pattern/PractitionerPatterns.java

```diff
@@ -0,0 +1,12 @@
+package health.ere.ps.service.muster16.parser.rgxer.delegate.pattern;
+
+import java.util.regex.Pattern;
+
+public class PractitionerPatterns extends Patterns {
+    public final Pattern NAME_PREFIX = Pattern.compile("(Dr)\\.");
+    public final Pattern NAME_LINE = Pattern.compile("(Dr\\.)*([a-z A-Z]+)(-){0,1}([a-z A-Z]+)");
+    public final Pattern STREET_LINE = Pattern.compile("^[a-z A-Zß]+(\\.{1})?.*[0-9]+");
```

Comment:
There are German characters that won't be captured by the letter pattern. You can use \\w instead.

Reply:
\w actually matches [A-Za-z0-9_], not any German character. I've added äöüÄÖÜß to the list of accepted letters, that will do the trick :) 
I don't think we should match whitespace here as such character would involve an error or weird behavior in our parsing.